src/Pos/Wallet/WalletMode.hs

```diff
@@ -152,10 +152,10 @@ instance MonadIO m => MonadTxHistory (WalletDB m) where
     getTxHistory addr _ = do
         chain <- WS.getBestChain
         utxo <- WS.getOldestUtxo
-        res <- fmap (fst . fromMaybe (panic "deriveAddrHistory: Nothing")) $
+        _ <- fmap (fst . fromMaybe (panic "deriveAddrHistory: Nothing")) $
             runMaybeT $ flip runUtxoStateT utxo $
             deriveAddrHistory addr chain
-        pure undefined
+        pure $ panic "undefined"
```

Comment:
I don't use `master`, it `cardano-sl-0.2` branch. Fix for `master` will be soon.